internal/access/access_key.go

```diff
@@ -20,12 +20,8 @@ func ListAccessKeys(c *gin.Context, identityID uid.ID, name string, showExpired
 	opts := data.ListAccessKeyOptions{
 		Pagination:     p,
 		IncludeExpired: showExpired,
-	}
-	if identityID != 0 {
-		opts.ByIssuedForID = identityID
-	}
-	if name != "" {
-		opts.ByName = name
+		ByIssuedForID:  identityID,
+		ByName:         name,
```

Comment:
I think this implies ByIssuedForID and ByName are no longer required. We're definitely going to break the security in a bad way if all required fields become optional

Reply:
These were previously optional: https://github.com/infrahq/infra/blob/v0.15.1/internal/access/access_key.go#L22-L23 

So I don't think this changes behaviour.

It's has always been up to the caller to decide if the parameter should be optional or not. Now we're doing that with an `if` statement in the caller, instead of a helper function that hides the `if`.

There aren't a lot of examples of that because generally the IDs are already required by API request validation.